src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_hana_commands.py

```diff
@@ -0,0 +1,558 @@
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+
+from azure.cli.testsdk import ScenarioTest, record_only
+import json
+import os
+
+
+id_hana = '/subscriptions/38304e13-357e-405e-9e9a-220351dcce8c/resourceGroups/SAPHANA-CLITEST-RG/providers/Microsoft.Compute/virtualMachines/saphana-clitest-vm'
+item_id_hana = '/subscriptions/38304e13-357e-405e-9e9a-220351dcce8c/resourceGroups/saphana-clitest-rg/providers/Microsoft.RecoveryServices/vaults/saphana-clitest-vault/backupFabrics/Azure/protectionContainers/VMAppContainer;compute;saphana-clitest-rg;saphana-clitest-vm/protectedItems/SAPHanaDatabase;hdb;hdb'
+sub_hana = '38304e13-357e-405e-9e9a-220351dcce8c'
```

Comment:
Why these resources can't be created dynamically in those tests, but use the fixed values? These subscription seems to be inaccessible to our team

Reply:
Creating HANA VM and registering it to azure backup services is a lengthy multi-step process that involves logging into the VM and running some scripts. I am not sure if this whole process can be automated using the command line. This will require an investigation of its own. We can look into the possibilities and probably take it up as a work item in upcoming releases if required.

Just wanted to understand, is it encouraged to use a dynamic setup over an existing setup for testing in CLI? We have been using this existing setup approach for testing on SQL and HANA VMs from the beginning and I don't remember getting an ask from the team to shift to a dynamic setup.

Reply:
Yes, we recommend creating resources dynamically in the tests, because this can avoid the problem that some fixed resources cannot be accessed by our team, and also ensure that the execution of the tests will not be affected after those test resources are deleted

Reply:
This PR can be merged first. You can submit a new separate PR to refactor the tests with these fixed resources in the future